launcher/res/Strings.ja-jp.xaml

```diff
@@ -0,0 +1,173 @@
+    <sys:String x:Key="Strings.MapCompression">圧縮されたエリアマップ</sys:String>
+    <sys:String x:Key="Strings.Mipmapping">遠距離表現を精細にする</sys:String>
+    <sys:String x:Key="Strings.Mipmapping0">なし</sys:String>
+    <sys:String x:Key="Strings.Mipmapping1">低</sys:String>
+    <sys:String x:Key="Strings.Mipmapping2">中</sys:String>
+    <sys:String x:Key="Strings.Mipmapping3">高</sys:String>
+    <sys:String x:Key="Strings.MiscellaneousSettings">その他</sys:String>
+    <sys:String x:Key="Strings.Save">保存</sys:String>
+    <sys:String x:Key="Strings.SupersampleAntiAliasing">アンチエイリアシング</sys:String>
+    <sys:String x:Key="Strings.SupersampleAntiAliasingFormat">{0:0.##×}</sys:String>
+    <sys:String x:Key="Strings.TextureCompression">テクスチャ品質</sys:String>
+    <sys:String x:Key="Strings.TextureCompressionHigh">低</sys:String>
+    <sys:String x:Key="Strings.TextureCompressionLow">標準</sys:String>
+    <sys:String x:Key="Strings.TextureCompressionUncompressed">高品質</sys:String>
+    <sys:String x:Key="Strings.UIScaleFormat">{0:0%}</sys:String>
+    <sys:String x:Key="Strings.AdjustGamma">明るさ調整</sys:String>
+    <sys:String x:Key="Strings.AdjustPosition">ウィンドウ位置を調整</sys:String>
+    <sys:String x:Key="Strings.Debug">デバッグコンソール</sys:String>
+    <sys:String x:Key="Strings.ChangeSettingsPath">設定保存先を上書き</sys:String>
+    <sys:String x:Key="Strings.ChangeUserPath">ユーザデータ保存先を上書き</sys:String>
+    <sys:String x:Key="Strings.ChangeTempPath">一時データ保存先を上書き</sys:String>
+    <sys:String x:Key="Strings.SelectSettingsPath">設定保存先を選択してください</sys:String>
+    <sys:String x:Key="Strings.SelectUserPath">ユーザデータ保存先を選択してください</sys:String>
+    <sys:String x:Key="Strings.SelectTempPath">一時データ保存先を選択してください</sys:String>
+    <sys:String x:Key="Strings.Browse">参照</sys:String>
+    <sys:String x:Key="Strings.DriverStability">ビデオカードの安定性を改善</sys:String>
+    <sys:String x:Key="Strings.FontType">フォント</sys:String>
+    <sys:String x:Key="Strings.FontTypeCompressed">標準</sys:String>
+    <sys:String x:Key="Strings.FontTypeHighQuality">高品質</sys:String>
+    <sys:String x:Key="Strings.FontTypeUncompressed">非圧縮</sys:String>
+    <sys:String x:Key="Strings.GammaFormat">{0:0.0}</sys:String>
+    <sys:String x:Key="Strings.Horizontal">水平位置</sys:String>
+    <sys:String x:Key="Strings.MaxSounds">効果音同時発音数</sys:String>
+    <sys:String x:Key="Strings.MaxSoundsFormat">{0:0;;Sound disabled}</sys:String>
+    <sys:String x:Key="Strings.PlayIntro">オープニングムービーを再生</sys:String>
+    <sys:String x:Key="Strings.PlaySoundWhenUnfocused">非アクティブ状態でサウンドを再生する</sys:String>
+    <sys:String x:Key="Strings.Vertical">垂直位置</sys:String>
+    <sys:String x:Key="Strings.ElevationRequired">PlayOnline Viewerを実行するには管理者権限が必要です。</sys:String>
+    <sys:String x:Key="Strings.AdjustAccessControl">権限の調整</sys:String>
+    <sys:String x:Key="Strings.FixAccessControlPrompt">管理者権限が必要です</sys:String>
+    <sys:String x:Key="Strings.FixAccessControlDescription">現在、PlayOnline Viewerを実行するには管理者権限が必要です。PlayOnline Viewerが管理者権限なしで実行できるようにしますか？</sys:String>
```

Comment:
If possible, could you add that Windower will adjust POL so admin privileges are no longer required?

Reply:
ok, I will some words.